.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rubyversion: ['2.4', '2.5', '2.6', '2.7', '3.0']
+        rubyversion: ['2.5', '2.6', '2.7']
```

Comment:
why stop testing on 3.0? did something break?

Reply:
This was actually just me sputtering 

![image](https://user-images.githubusercontent.com/79638/129612586-a7005c50-b16c-4e2a-8209-56030d3baef8.png)

can't get the 2.4 and 3.0 units to run, and seems modifications to this file don't help.

Reply:
Ah interesting, restoring the original file caused those units to run.